chidioguejiofor/airtech-api
# #165386785: Host app on heroku

#### What does this PR achieve?
- adds development and production environment for docker
- hosts app on Heroku on the [https://airtech-api.herokuapp.com](API Link)

#### Any background context?
- The app was hosted on Heroku using Docker

#### New environmental Variables?
- None

#### Manual Testing Steps
- Open your browser
- Go to the URL https://airtech-api.herokuapp.com
- You would see the `Welcome to API` message
